examples/apps/echo/client/echo_client.py

```diff
@@ -20,15 +20,18 @@
 import logging
 import base64
 import secrets
+import time
```

Comment:
Just as a warning, there will be a merge conflict with my minor change to `echo_client.py` that was integrated today (PR #56 ). 
Add the line `#! /usr/bin/env python3` at the beginning and make the file executible (`chmod +x echo_client.py`).

Reply:
sure Dan. I will take care of merge conflicts.